### PR TITLE
fix: correct trailing slash for api docs endpoints and update tests

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
Currently only works https://localhost:8443/api/eda/v1/docs, docs/ returns 404
This PR fixes the correct handling of the trailing slash to keep it consistent with the rest of the application, so /docs is a 301 to -> /docs/ and /docs/ works as expected
Also adds sanity tests for all these urls.

### CODE REVIEW

**Review by Dostonbek1 — APPROVED** [review]
I observed this a while ago but never got around to pushing a fix for it. Thank you for putting up the PR. 👍

File: tests/integration/api/test_openapi.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "path",
+    [
+        pytest.param("/openapi.json", id="openapi_json"),
+        pytest.param("/openapi.yaml", id="openapi_yaml"),
+    ],
+)
+@pytest.mark.django_db
+def test_v1_openapi(admin_client, path):
+    response = admin_client.get(f"{api_url_v1}{path}")
+    assert response.status_code == 200
```
Comment (jshimkus-rh) [inline tests/integration/api/test_openapi.py:35]: I'd prefer the use of the `status` value from `rest_framework` over a hardcoded value.

File: tests/integration/api/test_openapi.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+@pytest.mark.parametrize(
+    "path",
+    [
+        pytest.param("/docs/", id="swagger_ui"),
+        pytest.param("/redoc/", id="redoc"),
+    ],
+)
+def test_v1_openapi_ui(admin_client, path):
+    response = admin_client.get(f"{api_url_v1}{path}")
+    assert response.status_code == 200
```
Comment (jshimkus-rh) [inline tests/integration/api/test_openapi.py:50]: I'd prefer the use of the `status` value from `rest_framework` over a hardcoded value.

File: tests/integration/api/test_openapi.py (hunk 1 above)
Comment (Alex-Izquierdo) [reply tests/integration/api/test_openapi.py]: changed

File: tests/integration/api/test_openapi.py (hunk 2 above)
Comment (Alex-Izquierdo) [reply tests/integration/api/test_openapi.py]: changed

**Review by jshimkus-rh — APPROVED** [review]

**Pull request merged**